src/models/shape_loader.cpp

```diff
@@ -636,6 +636,10 @@ bool readPose(tue::config::Reader& cfg, geo::Pose3D& pose, tue::config::Required
         else
             return false;
     }
+    else
```

Comment:
I don't know where the changes in this file come from and how they're related to https://github.com/tue-robotics/rgbd/pull/26/files

Reply:
These changes shouldn't be here. I will revert those.